pom.xml

```diff
@@ -959,5 +959,7 @@
     <versions-maven-plugin.version>2.5</versions-maven-plugin.version>
     <xml-maven-plugin.version>1.0</xml-maven-plugin.version>
     <maven-license-plugin.version>1.7</maven-license-plugin.version>
+    <incrementals-maven-plugin.version>1.0-beta-5</incrementals-maven-plugin.version>
+    <incrementals-enforce-minimum.version>1.0-beta-4</incrementals-enforce-minimum.version>
```

Comment:
Extracting to properties make it easier to read what happens. Question becomes should they follow each other.